src/components/Quotes.jsx

```diff
@@ -0,0 +1,51 @@
+import { useState, useEffect } from 'react';
+import axios from 'axios';
+
+const Quotes = () => {
+  const [quote, setQuote] = useState('');
+  const [author, setAuthor] = useState('');
+  const [loading, setLoading] = useState(false);
+  const [error, setError] = useState('');
+
+  useEffect(() => {
+    const category = 'happiness';
+
+    setLoading(true);
+    axios.get('https://api.api-ninjas.com/v1/quotes', {
+      params: { category },
+      headers: {
+        'X-Api-Key': 'HmE/CVXQyHmYONdR03v5PA==RVOBKSCgWyZif3BP',
+      },
+    })
+      .then((response) => {
+        const data = response.data[0];
+        setQuote(data.quote);
+        setAuthor(data.author);
+        setLoading(false);
+      })
+      .catch(() => {
+        setError('Error displaying quates');
+        setLoading(false);
+      });
+  }, []);
+
+  if (loading) {
+    return <p>Loading...</p>;
```

Comment:
- [ ] [OPTIONAL] I have noticed an error coming from the API, I'm sure this is something to do with the key. Kindly fix this error so that the app works as expected.

![cors](https://user-images.githubusercontent.com/53124548/227459627-27192d20-1fc1-477d-a028-c6b143f0f067.png)